Test.Integration.Net7/EventsAPI/CreateContents.cs

```diff
@@ -0,0 +1,433 @@
+using Sift;
+using System.Collections.ObjectModel;
+using Test.Integration.Net7.Uitlities;
+using Xunit;
+
+namespace Test.Integration.Net7.EventsAPI
+{
+    public class CreateContents
```

Comment:
Let's rename to `ContentTest` and add at least one test case for Update Content event (for a listing, as an example)